CODEOWNERS

```diff
@@ -63,4 +64,4 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 /ngraph/  @openvinotoolkit/openvino-ngraph-maintainers
 
 # Tools
-/tools/  @openvinotoolkit/openvino-tools-maintainers
+/tools/  @openvinotoolkit/openvino-tools-maintainers @akuporos
```

Comment:
you is a member of @openvinotoolkit/openvino-tools-maintainers now

Reply:
fixed